fdbserver/BlobWorker.actor.cpp

```diff
@@ -5627,31 +5627,19 @@ ACTOR Future<Void> blobWorker(BlobWorkerInterface bwInterf,
 			printf("Initializing blob worker s3 stuff\n");
 		}
 
-		try {
```

Comment:
github's diff doesn't like this, but all this does is ensure all of this logic is in the try-catch that sends recruitment_failed back to the BM on error

Reply:
Github "Split View" may look better.